pom.xml

```diff
@@ -559,7 +559,7 @@
             <dependency>
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
-                <version>6.0.6</version>
+                <version>8.0.12</version>
```

Comment:
why update the version? We are on mysql-5.7 afair.

Reply:
MySQL Connector/J 8.0 is highly recommended for use with MySQL Server 8.0, 5.7, 5.6, and 5.5